retroshare-gui/src/retroshare-gui.pro

```diff
@@ -1322,6 +1324,7 @@ gxsforums {
 		gui/gxsforums/GxsForumThreadWidget.cpp \
 		gui/gxsforums/GxsForumModel.cpp \
 		gui/gxsforums/GxsForumUserNotify.cpp \
+		gui/gxsforums/ForumMessageView.cpp \
```

Comment:
Why didn't you named it GxsForumMessageView or best GxsForumMessageDialog ?

Reply:
i can rename it, but i never like add naming with GXS before , normaly on gui we doesnt need everything naming with gxs? only to look same when other classes starts with gxs right..?

Reply:
Neither do I, but all other class are named like this so keep homogeneity.

Reply:
ok

Reply:
> Neither do I, but all other class are named like this so keep homogeneity.

hi i has renamed it, if you see some other things?

Reply:
Yes some little, I'll comment them latter. I don't have the time now.